doc/source/development.rst

```diff
@@ -99,6 +99,47 @@ directory will take effect without reinstalling the package.
 
 .. warning:: if you run ``python setup.py install``, files will be copied from the Ray directory to a directory of Python packages (``/lib/python3.6/site-packages/ray``). This means that changes you make to files in the Ray directory will not have any effect.
 
+Building Ray on Windows (full)
+------------------------------
+
+**Requirements**
+
+- Microsoft Visual Studio 2019 (or Microsoft Build Tools 2019)
+- JDK 15
+- Miniconda 3
```

Comment:
Point out this should be installed to `C:\ProgramData\Miniconda3` or the paths below need to be changed

Reply:
Added more details to the instructions.